IDE/ECLIPSE/DEOS/deos_malloc.c

```diff
@@ -19,27 +19,11 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1335, USA
  */
 #include <wolfssl/wolfcrypt/settings.h>
-#include <wolfssl/ssl.h>
-
-#define ROUND_UP(x, align)  (((int) (x) + (align - 1)) & ~(align - 1))
-#define SIZEOF_HEADER sizeof(size_t)  /* tracks size of allocated block */
-
-#define HEAP_SIZE_MAX   (1*1024*1024)
-
-static size_t allocatedMemory = 0;
-
-size_t getMemAllocatedSize_deos(size_t* size){
-
-    if (size)
-        *size = allocatedMemory;
-
-    return allocatedMemory;
-}
-
-/* Simply returns without freeing any memory. */
+#include <wolfssl/wolfcrypt/types.h>
+#include <deos.h>
 
 void free_deos(void *ptr) {
-    //printf("fake free_deos()\n");
+    free(ptr);
```

Comment:
Since Deos added the ansi library starting from version 4.7, it make sense to use malloc/free of the library instead of using these custom implementations.

Reply:
They are still needed since `realloc` is not available.